vendor/src/github.com/docker/notary/client/delegations.go

```diff
@@ -0,0 +1,294 @@
+package client
+
+import (
+	"encoding/json"
```

Comment:
@riyazdf this looks dangerous to me. Shouldn't we use the safer json? By the way it'd be great to rebase that package on top of go1.6 too